### ISSUE TITLE
Reconsider rebranding Polkadot.js to remove the "Polkadot"

### ISSUE THREAD

**Issue by csaint02** [polkadot-js/apps#9189]
There has been a lot of conversation in the Polkadot community about rebranding Polkadot.js to Substrate Developer Console, or honestly anything separate from the "Polkadot" brand.

I am not a dev, but I do know that .js is a crucial tool within the Substrate developer ecosystem which continues to grow over time. I think the most important motivation for this rebrand is to stop confusing new users that are onboarding to Polkadot and innately feel the need to use .js as their wallet because of the Polkadot name and the long standing history

Over the past few years there have been multiple wallets emerge that cater to the user rather than the dev. And in my humble opinion, I think the devs will find this tool through word of mouth or necessity and it would be much less of a problem than the converse scenario. In this respect, I am humbly asking for the maintainer to consider changing the name. There have been a number of people commenting on this subject as of late both from the dev community and the user community.

Thank you and please consider these links as reference for the community support for a rebrand. I am happy to supply more and encourage people to comment here to show support. 

I would also ask, what would it take for you to seriously consider this rebrand?

https://forum.polkadot.network/t/polkadot-js-apps-messaging-rebranding-thoughts/2001/18

https://twitter.com/LeemoXD/status/1635953191765540865?s=20

https://twitter.com/csaint02/status/1636343931544764416?s=20

**Comment by lolmcshizz** [comment]
Also replace the old (outdated!) Polkadot logo on the extension

<img width="56" alt="image" src="https://user-images.githubusercontent.com/97540577/225914563-0ecdb24d-a04e-412d-8f38-3670a7db1809.png">

Replace with substrate logo
![image](https://user-images.githubusercontent.com/97540577/225914502-21cca496-399f-4022-90ef-aae6e7bf6db3.png)

**Comment by rossbulat** [comment]
Substrate Developer Console or <active chain> Developer Console sounds reasonable to me.

**Comment by csaint02** [comment]
Even just substrate.js would be a good middle ground

**Comment by csaint02** [comment]
@jacogr any thoughts on the matter? Sorry to tag, I am sure you're busy, but I think many are curious your thoughts.

**Comment by arrudagates** [comment]
> Substrate Developer Console

I agree with rebranding to Substrate Developer Console, and furthermore I strongly believe the rest of the API should follow on the branding -> substrate.js API, Substrate Developer Extension, etc.

**Comment by rossbulat** [comment]
@jacogr are you able to give some feedback.

**Comment by ltfschoen** [comment]
It makes sense, maybe similar to why they split https://github.com/paritytech/polkadot into --> paritytech/substrate & paritytech/cumulus & paritytech/polkadot

If we actually went down this path we'd probably need to split it into a Substrate API and a Polkadot API.

We'd have to move individual parachain aspects from each repo to the Polkadot API.

We'd have to do things like:
* split https://github.com/polkadot-js/apps repo into --> substrate-js/apps & polkadot-js/apps
* split https://www.npmjs.com/package/@polkadot/apps into --> @substrate/apps & @polkadot/apps
* split https://github.com/polkadot-js/api repo into --> substrate-js/api & polkadot-js/api
* split https://www.npmjs.com/package/@polkadot/api into --> @substrate/api & @polkadot/api
* split all the other @polkadot NPM libraries listed here https://www.npmjs.com/search?q=%40polkadot into --> @substrate/____ and @polkadot/____
* split docs --> https://substrate.js.org & https://polkadot.js.org
* split https://github.com/polkadot-js/extension --> substrate-js/extension & polkadot-js/extension
* split https://github.com/polkadot-js/tools --> substrate-js/tools & polkadot-js/tools

We'd have to split the Polkadot Wiki https://wiki.polkadot.network/docs/polkadotjs into --> Substrate Wiki & Polkadot Wiki

Are there any market research findings on what proportion of new users create their wallets by using Polkadot.js versus other means like following the Substrate documentation branding by using Subkey https://docs.substrate.io/reference/command-line-tools/subkey/ ?   And findings on what proportion of those users get confused and during the process and what are the different UI-specific causes behind that?  What proportion don't end up converting into users of the ecosystem as a result? 

With regards to the new wallets that are emerging, many of them aren't open-source or used to be but are now closed-source, so as a dev I still use Polkadot.js for pretty much everything.  Polkadot.js has the most eyes on it and I'd argue is the most trusted. It even warned users like myself to be wary of contributing to crowdloans using other wallet interfaces, since some campaigns websites didn't always deposit directly to the crowdloan module and instead deposited to their own multisig, which in some instances was used to cause delays in returning deposited crowdloan tokens to users.
